packages/wallet/backend/tests/rafiki/service.test.ts

```diff
@@ -0,0 +1,51 @@
+import { Bindings } from '@/app'
+import { env } from '@/config/env'
+import { createContainer } from '@/createContainer'
+import { EventType, RafikiService } from '@/rafiki/service'
+import { Container } from '@/shared/container'
+import { Knex } from 'knex'
+import { TestApp, createApp } from '../app'
+import { truncateTables } from '../tables'
+import { mockOutgoingPaymenteCreatedEvent } from '../mocks'
+
+describe('Rafiki Service', () => {
+  let bindings: Container<Bindings>
+  let knex: Knex
+  let rafikiService: RafikiService
+  let appContainer: TestApp
+
+  beforeAll(async (): Promise<void> => {
+    bindings = createContainer(env)
+    appContainer = await createApp(bindings)
+    knex = appContainer.knex
+    rafikiService = await bindings.resolve('rafikiService')
+  })
+
+  afterAll(async (): Promise<void> => {
+    appContainer.stop()
+    knex.destroy()
+  })
+
+  afterEach(async (): Promise<void> => {
+    await truncateTables(knex)
+  })
+
+  describe('OnWebHook', () => {
+    it('should throw an error if the paymentPointer is invalid', async () => {
+      const webHook = mockOutgoingPaymenteCreatedEvent({})
+
+      await expect(rafikiService.onWebHook(webHook)).rejects.toThrowError(
+        /Invalid payment pointer/
+      )
+    })
+
+    it('should throw an error unknow event type mock-event', async () => {
+      // eslint-disable-next-line no-use-before-define
+      const webHook = mockOutgoingPaymenteCreatedEvent({type: "mock-event"})
```

Comment:
@Tymmmy CAn you see it here. Is doesn't work.

Reply:
Comment out the whole testcase, leave it there with a TODO comment above the function, and when it is merged into master we will fix it, when I can get the code to show the error for me as well :)

Reply:
Done. Check it out ?